python/pyiceberg/cli/console.py

```diff
@@ -286,7 +286,7 @@ def get_table(ctx: Context, identifier: str, property_name: str):
 
 @properties.group()
 def set():
-    """Removes properties on tables/namespaces"""
+    """Sets a property on tables/namespaces"""
```

Comment:
should we update this line as well : 
[1] https://github.com/apache/iceberg/pull/6372/files#diff-6a69efbab813860a0fb4f238e55a67bc64c3fe5c307647faca64a23e2a8dc875R299
 to """Sets a property of a namespace"""